### DISCUSSION TITLE
Hotend temperature limiting

### DISCUSSION THREAD

**Discussion by mosheas** [Jyers/Marlin#1803] [Q&A]
Where can I edit the firmware for max hotend temperature  limiting?

**Comment by bendead** [comment]
In the Configuration.h file which is inside the Marlin or from the Configuration Files folder

You need to add 15 to the temperature you want, for exemple here it's 260 max

#define HEATER_0_MAXTEMP 275
#define HEATER_1_MAXTEMP 275
#define HEATER_2_MAXTEMP 275
#define HEATER_3_MAXTEMP 275
#define HEATER_4_MAXTEMP 275
#define HEATER_5_MAXTEMP 275
#define HEATER_6_MAXTEMP 275
#define HEATER_7_MAXTEMP 275

Be carful, if you did not upgrade your hotend, the PTFE tube inside the stock one will melt above that and even can happen bellow

**Reply by mosheas** [reply]
So if I got it right, I have to change each line to a value of 290?
And then what would be the maximum temperature? 
Thank you.

**Comment by bendead** [comment]
Yes, if you change them all to 290 the max temp selectable on the printer will be 275

**Reply by mosheas** [reply]
👍 Thank you
